### PR TITLE
Update llb-api-spec.md

### PR DESCRIPTION
**Pull request by anirudh723**
Updated the GET and POST requests regarding announcements, to have the optional imageSrc field. Apparently the eventId was also never updated in the docs so added that too.

### CODE REVIEW

File: docs/llb/llb-api-spec.md [hunk 1]
```diff
@@ -705,6 +707,8 @@ The announcements were retrieved successfully.
       "id": ID,
       "title": STRING,
       "description": STRING,
+      "eventId": ID OR NULL,
```
Comment (jackblanc) [inline docs/llb/llb-api-spec.md:710]: I believe that this route does not include the `eventId`, as it gets site-wide announcements that are not associated with an event. If you observed something different, what `eventId` is returned?

File: docs/llb/llb-api-spec.md [hunk 2]
```diff
@@ -662,6 +662,8 @@ The announcements were retrieved successfully.
...
```
Comment (jackblanc) [inline docs/llb/llb-api-spec.md:665]: Same comment here - are you sure `eventId` is included?

File: docs/llb/llb-api-spec.md [hunk 3]
```diff
@@ -749,18 +754,17 @@ The announcements were retrieved successfully.
...
 
 ```json
 {
-    "announcement": {
-        "title": STRING,
-        "description": STRING
-    }
+    "title": STRING,
+    "description": STRING,
+    "imageSrc": STRING
```
Comment (jackblanc) [inline docs/llb/llb-api-spec.md:767]:
```suggestion
    "imageSrc": STRING OR NULL
```

Comment (jackblanc) [reply docs/llb/llb-api-spec.md:767]: imageSrc is optional, as you say above

File: docs/llb/llb-api-spec.md [hunk 4]
```diff
@@ -798,7 +804,8 @@ Body:
 ```json
 {
     "title": STRING,
-    "description": STRING
+    "description": STRING,
+    "imageSrc": STRING
```
Comment (jackblanc) [inline docs/llb/llb-api-spec.md:808]:
```suggestion
    "imageSrc": STRING OR NULL
```

**Review by jackblanc — CHANGES_REQUESTED** [review]
There are a few things I'm not sure about.

File: docs/llb/llb-api-spec.md (hunk 1 above)
Comment (anirudh723) [reply docs/llb/llb-api-spec.md:710]: Yeah it does:
image

File: docs/llb/llb-api-spec.md (hunk 2 above)
Comment (anirudh723) [reply docs/llb/llb-api-spec.md:665]: Yeah I guess it does. See [comment](https://github.com/Code-4-Community/c4c-dev-docs/pull/48/files/68cd8276192699d76b853e19c9e7ff1d0fc7b3a2#r586929153)

**Review by jackblanc — APPROVED** [review]
Is it always NULL? Seems odd but okay

**Pull request merged**